docs/functions.md

```diff
@@ -3,26 +3,30 @@
 Currently we have almost 15+ functions in our npm package. Some of them are listed as below.
 
 - Arithmetic Functions
-    * [doAdd()](aFunc/doAdd.md):- Add 2 numbers
-    * [doAddN()](aFunc/doAddN.md):- Add many numbers
-    * [doSub()](aFunc/doSub.md):- Subtract 2 numbers
-    * [doPositiveSub()](aFunc/doPositiveSub.md):- Give positive result after subtraction.
-    * [doMul()](aFunc/doMul.md):- Multiply 2 numbers
-    * [doMulN()](aFunc/doMulN.md):- Multiply many numbers
-    * [doDiv()](aFunc/doDiv.md):- Divide 2 numbers
-    * [doInc()](aFunc/doInc.md):- Increment a number(+1)
-    * [doDecr()](aFunc/doDecr.md):- Decrement a number(-1)
+
+  - [doAdd()](aFunc/doAdd.md):- Add 2 numbers
+  - [doAddN()](aFunc/doAddN.md):- Add many numbers
+  - [doSub()](aFunc/doSub.md):- Subtract 2 numbers
+  - [doPositiveSub()](aFunc/doPositiveSub.md):- Give positive result after subtraction.
+  - [doMul()](aFunc/doMul.md):- Multiply 2 numbers
+  - [doMulN()](aFunc/doMulN.md):- Multiply many numbers
+  - [doDiv()](aFunc/doDiv.md):- Divide 2 numbers
+  - [doInc()](aFunc/doInc.md):- Increment a number(+1)
+  - [doDecr()](aFunc/doDecr.md):- Decrement a number(-1)
     <!-- * [toBin()](aFunc/toBin.md):- Decimal to Binary -->
     <!-- * [toHex()](aFunc/toHex.md):- Decimal to Hexadecimal -->
     <!-- * [toOct()](aFunc/toOct.md):- Decimal to Octal -->
     <!-- * [toTri()](aFunc/toTri.md):- Decimal to Trinary -->
-    * [doMod()](aFunc/doMod.md):- Get the modulus
+  - [doMod()](aFunc/doMod.md):- Get the modulus
 
 - Miscellaneous Functions
-    * [doReverse()](mFunc/doReverse.md):- Reverse number
-    * [calcAge()](mFunc/calcAge.md):- Calculate age
-    * [calcLoan()](mFunc/calcLoan.md):- Calculate total amount of loan
-    * [timeCalc()](mFunc/timeCalc.md):- Calculate time
-    * [getCurrency()](mFunc/getCurrency.md):- Convert Currency
+  - [doMax()](mFunc/doMax.md):- Maximum element of Array
+  - [doMin()](mFunc/doMin.md):- Minimum element of Array
+  - [doFactorial()](mFunc/doFactorial.md):- Factorial of a number
+  - [doReverse()](mFunc/doReverse.md):- Reverse number
+  - [calcAge()](mFunc/calcAge.md):- Calculate age
+  - [calcLoan()](mFunc/calcLoan.md):- Calculate total amount of loan
+  - [timeCalc()](mFunc/timeCalc.md):- Calculate time
+  - [getCurrency()](mFunc/getCurrency.md):- Convert Currency

```

Comment:
The same thing needs to be applied here too.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
mFunc/Filename is the path right so do I need to change to mFunction/doMax. Because the path mFunction/doMax doesnot exists right?

Reply:
please give an example

Reply:
> mFunc/Filename is the path right so do I need to change to mFunction/doMax. Because the path mFunction/doMax doesnot exists right?

Yes, you're right.

Reply:
- [doMax()](mFunc/doMax.md):- Maximum element of Array
Tell me how I should change above line?

Reply:
> * [doMax()](mFunc/doMax.md):- Maximum element of Array
>   Tell me how I should change above line?

Give a screenshot.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
intend fixed please review